test_opensearchpy/test_async/test_connection.py

```diff
@@ -97,6 +97,27 @@ def test_opaque_id(self):
         con = AIOHttpConnection(opaque_id="app-1")
         assert con.headers["x-opaque-id"] == "app-1"
 
+    def test_trust_env(self):
```

Comment:
Add a test for defaults if we don't have one yet, `test_async_opensearch_default_options`, and one that changes all the options `test_async_opensearch_options`.